src/content/docs/release-notes/docs-release-notes/docs-9-27-2024.mdx

```diff
@@ -0,0 +1,48 @@
+---
+subject: "Docs"
+releaseDate: '2024-09-27'
+version: 'version: September 20-26, 2024'
+---
+
+### New docs
+
+* Our new [DataBricks integration](/docs/infrastructure/other-infrastructure-integrations/databricks-integration/) describes how to use this integration to collect telemetry from Spark deployments running on Databricks or elsewhere.
```

Comment:
"integration describes how to use this integration", it's probably not a big deal but that is two of the same word really close together

Reply:
You're right! Thanks for catching that!